site/docs/relations/physical_relations.md

```diff
@@ -47,7 +47,45 @@ The nested loop join operator does a join by holding the entire right input and
+| Outputs              | 1                                                                                            |
+| Property Maintenance | It is the same as the [Hash Equijoin Operator](physical_relations.md#hash-equijoin-operator) |
+| Direct Output Order  | Same as the [Join](logical_relations.md#join-operator) operator.                             |
+
+### Duplicate Eliminated Join Properties
+
+| Property                  | Description                                                                                                                                                                                                             | Required                 |
+|---------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Left Input                | A relational input.                                                                                                                                                                                                     | Required                 |
+| Right Input               | A relational input.                                                                                                                                                                                                     | Required                 |
+| Left Keys                 | References to the fields to join on in the left input.                                                                                                                                                                  | Required                 |
+| Right Keys                | References to the fields to join on in the right input.                                                                                                                                                                 | Required                 |    
+| Post Join Predicate       | An additional expression that can be used to reduce the output of the join operation post the equality condition. Minimizes the overhead of secondary join conditions that cannot be evaluated using the equijoin keys. | Optional, defaults true. |
+| Join Type                 | One of the join types defined in the Join operator.                                                                                                                                                                     | Required                 |
+| Duplicate Eliminated Side | The side that is deduplicated and pushed into the other side.                                                                                                                                                           | Required                 |
```

Comment:
Why does this need to be specified?  Isn't it always the build side (which is determined by the join type already)?

Reply:
That's a good point. I know in substrait we typically have left and right types for the same join type, so I think you are right.

@lnkuiper, just to double-check the sideness in the join type is sufficient to remove the duplicate eliminated side?